web/modules/custom/os2forms_user_menu/src/Plugin/Block/UserMenuBlock.php

```diff
@@ -145,15 +144,19 @@ public function build() {
       return [];
     }
 
+    $sessionType = $webformNemIdSettings['session_type'] ?? NULL;
+
+    $plugin = $sessionType ? $this->authProvider->getPluginInstance($webformNemIdSettings['session_type']) : $this->authProvider->getActivePlugin();
```

Comment:
I think we have a typo here:

```suggestion
    $plugin = $sessionType ? $this->authProvider->getPluginInstance($sessionType) : $this->authProvider->getActivePlugin();
```